README.md

```diff
@@ -67,6 +67,8 @@ perl tools/builder/build_mfx.pl --cmake=intel64.make.release
 ```
 This will build MSDK binaries and MSDK samples.
 
+Please note build_mfx.pl optional flags, like one example of: --no-warn-as-error to disable default: treat Warning As Error.
```

Comment:
Please mention, that using this option may be required when using newer version (`>4.8`) of compiler (probably better place it after paragraph about using CLang compiler). And this option is `--no_warn_as_error`, with underscores, not dashes.